ee/localserver/krypto-ec-middleware.go

```diff
@@ -334,6 +344,14 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 
 		w.Header().Add(kolideKryptoHeaderKey, kolideKryptoEccHeader20230130Value)
 
+		for k, v := range bhr.Header() {
```

Comment:
These headers are outside the krypto box, is that okay?

Reply:
It would be better if they were signed, but also not fatal. I thought if we put them into the CmdReq they'd be inside?

Reply:
The headers of the request are inside the CmdReq (in krypto box). We don't have a mechanism currently to add headers from the response to the krypto box. Currently we just grab the body from the response and put it in the krypto box. We could come up with some kind of base response that we imbed in each of the local server responses.

Reply:
I'm confused. We have too many layers

I think it's important for the response from launcher to k2, to be in the box. Otherwise it's a bit too trivial to slap whatever you want in flight.

Reply:
But I'm not sure I'm concerned about what's happening between localserver and the desktop.